src/views/Teams/Teams.js

```diff
@@ -47,26 +47,39 @@ function Teams() {
                   <td>
                     <Link to={`/teams/${team.id}`}>{team.name}</Link>
                   </td>
-                  <td>
-                    <Link to={`/teams/${team.id}`}>
-                      <button type="button" className="btn-view">
-                        View
-                      </button>
-                    </Link>
 
-                    <Link to={`/teams/${team.id}/edit`}>
-                      <button type="button" className="btn-edit">
+                  {user && (
+                    <td>
+                      <Link to={`/teams/${team.id}`}>
+                        <button type="button" className="btn-view">
+                    View
+                        </button>
+                      </Link>
+                      <Link to={`/teams/${team.id}/edit`}>
+                        <button type="button" className="btn-edit">
                         Edit
-                      </button>
-                    </Link>
-                    <button type="button" className="btn-delete" onClick={() => handleDelete(team)}>
+                        </button>
+                      </Link>
+                      <button type="button" className="btn-delete" onClick={() => handleDelete(team)}>
                       Delete
-                    </button>
-                  </td>
+                      </button>
+                    </td>)}
+                  {!user && (
```

Comment:
No need to write this `td` out twice, we could have the view with no conditions, and keep the edit / delete conditional